src/Serializers/SnakSerializer.php

```diff
@@ -81,6 +82,12 @@ private function getSerialized( Snak $snak ) {
 			$serialization['datavalue'] = $this->dataValueSerializer->serialize( $snak->getDataValue() );
 		}
 
+		if ( $snak instanceof DerivedPropertyValueSnak ) {
+			foreach ( $snak->getDerivedDataValues() as $key => $dataValue ) {
+				$serialization['derived-values'][$key] = $this->dataValueSerializer->serialize( $dataValue );
```

Comment:
I know there is no error, though should this not be done?

`$serialization['derived-values'] = array();`